ad-first-campaign.py

```diff
@@ -0,0 +1,66 @@
+def main():
+
+    import sem
+    import math
+    import numpy as np
+    import os
```

Comment:
maybe
```suggestion
    from os import path
```
will suffice